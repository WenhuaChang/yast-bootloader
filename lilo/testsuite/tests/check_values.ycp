{
    include "testsuite.ycp";
    include "lilo/routines.ycp";

    TEST(``(check_values($["label":"A", "alias": "B"])), [], nil);
    TEST(``(check_values($["label": "", "alias": "A"])), [], nil);
    TEST(``(check_values($["prompt": "", "timeout": 0])), [], nil);
    TEST(``(check_values($["disk": "disk=/dev/hda,bios=0x80", "change-rules":""])), [], nil);
    TEST(``(check_values($[])), [], nil);

}
