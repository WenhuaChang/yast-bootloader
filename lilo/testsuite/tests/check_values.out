Return	[]
Return	["label"]
Return	[]
Return	["change-rules"]
Return	[]
